src/_posts/addons/scalingo-docker-image/2000-01-01-start.md

```diff
@@ -1,47 +1,63 @@
 ---
 title: Scalingo Docker Image Addon
 nav: Introduction
-modified_at: 2022-07-19 00:00:00
+modified_at: 2022-09-07 00:00:00
 tags: docker images download feature
 index: 1
 ---
 
 ## Introduction
 
-Thanks to this addon, you'll be able to get back the **Docker Images** that have been built when deploying your application in our infrastructure.
+Thanks to this addon, you'll be able to get back the **Docker Images** that
+have been built when deploying your application in our infrastructure.
```

Comment:
IMO, the future tense is not relevant here.

Reply:
The whole phrase is not relevant, i removed it. There was 2 block saying the same things